techtalk_slides.tex

```diff
@@ -197,6 +210,8 @@ \section{Overview}
 \end{frame}
 
 \begin{frame}{Outline of talk}
+    % TODO this also includes "overview" and "conclusion",
+    % do we need this slide at all?
```

Comment:
The outline command just generates all of it automatically. I think I know how to prevent it from generating the overview and conclusion in the outline, if that's better.
I think it's helpful to give a quick roadmap of a talk though, and this was an easier way to do it than making a handmade list